stdlib/public/runtime/Casting.cpp

```diff
@@ -800,7 +798,7 @@ static bool _dynamicCastToExistential(OpaqueValue *dest,
     // A fallback to use if we don't have a more specialized approach
     // for a non-class type.
     auto fallbackForNonClass = [&] {
-#if SWIFT_OBJC_INTEROP
+#if SWIFT_OBJC_INTEROP // TODO
```

Comment:
The `// TODO`s are for the implementation of `_SwiftValue` (the second bullet in the PR description).

Reply:
This is done and the TODOs need to be removed.